music_shop/playlist/validation.py

```diff
@@ -0,0 +1,17 @@
+import copy
+
+from django.db.models import ObjectDoesNotExist
+
+from song.models import Song
+
+
+def validate_songs(request):
+    songs_pk = copy.deepcopy(request.data["song"])
+    request.data["song"].clear()
```

Comment:
Why should we do `deepcopy`?

Reply:
We should not use `deepcopy`. Difference between `deepcopy` and `copy` is seen only in nested objects.
So we can use `copy()` or `[:]`( We can use slice because `request.data["song"]` is a list)